docs/client_reference.rst

```diff
@@ -208,6 +209,16 @@ The client session supports the context manager protocol for self closing.
                          disabling.  See :ref:`aiohttp-client-tracing-reference` for
                          more information.
 
+   :param Callable[[ClientResponse,bytes],str] fallback_charset_resolver:
```

Comment:
Not sure if there is a way to get Sphinx to recognise this as the type while including spaces? Otherwise it takes the type as `Callable[[ClientResponse,` and the parameter name as `bytes, str] fallback_charset_resolver`...

Reply:
I'm going to merge, if someone has a fix, it can be done in a followup PR.

Reply:
@Dreamsorcerer I think that this should work
```rst
:type fallback_charset_resolver: Callable[[ClientResponse, bytes], str]
```

Reply:
That's not working, unfortunately.